kili/services/asset_import/base.py

```diff
@@ -14,9 +14,10 @@
 )
 from kili.helpers import format_result, is_url
 from kili.orm import Asset
+from tqdm import tqdm
+
 from kili.queries.asset import QueriesAsset
 from kili.utils import bucket, pagination
-from kili.utils.tqdm import tqdm
```

Comment:
you should rather update `kili/utils/tqdm.py` with `tqdm import tqdm` to keep things centralized.